dart-impl/mpi/src/dart_team_group.c

```diff
@@ -27,9 +27,10 @@
 
 static struct dart_group_struct* allocate_group()
 {
-  struct dart_group_struct* group = malloc(sizeof(struct dart_group_struct));
+  struct dart_group_struct* group = (struct dart_group_struct *) malloc(
```

Comment:
I thought that this is not necessary as well, but `clang-analyze` does not agree on that (all build flags we use are passed): `Unix API Allocator sizeof operand mismatch`. Anyway: Does this hurt?  We should at least use the cast always or never.